src/opf/opf.jl

```diff
@@ -1,10 +1,244 @@
 using MathOptSymbolicAD
+using SparseArrays
 
 mutable struct OPFModel{OPF <: PM.AbstractPowerModel}
     data::Dict{String,Any}
     model::JuMP.GenericModel
 end
 
+struct OPFData
+    case::String
+
+    N::Int  # number of buses
+    E::Int  # number of branches
+    G::Int  # number of generators
+    L::Int  # number of loads
+
+    Ag::SparseMatrixCSC{Float64,Int}  # generator incidence matrix
+
+    # Bus data
+    vmin::Vector{Float64}
```

Comment:
Add a `vnom` field that holds the nominal bus voltage
```suggestion
    # Bus data
    vnom::Vector{Float64}
    vmin::Vector{Float64}
```

This allows to run some analyses per voltage level (e.g. <200kV vs >= 200kV), which is not possible if one only has access to the per-unit values.